website/templates/emails/forward_invite.txt.mako

```diff
@@ -10,20 +10,15 @@ After ${fullname} confirms their account, they will be able to contribute to the
 
 Hello ${fullname},
 
-You have been added by ${referrer.fullname} as a contributor to project
-"${node.title}" on the Open Science Framework. To set a password for your account,
-visit:
+You have been added by ${referrer.fullname} as a contributor to the project "${node.title}" on the <a href="http://osf.io/">Open Science Framework</a>. To set a password for your account, visit:
```

Comment:
This file is for the text-only version of the email, and shouldn't have any HTML embedded at all. Feel free to copy it to `forward_invite.html.mako` and include links there.